flow/tutorials/TutorialScripts/drt/drc_fix.tcl

```diff
@@ -0,0 +1,35 @@
+source "helpers.tcl"
+
+read_lef ../../../platforms/sky130hd/lef/sky130_fd_sc_hd.tlef
+read_lef ../../../platformms/sky130hd/lef/sky130_fd_sc_hd_merged.lef
+read_def ./gcd/4_cts.def
+read_sdc ./gcd/gcd.sdc
+
+#global_route
+set_global_routing_layer_adjustment met1-met5 0.5
+set_routing_layers -signal met1-met5
+set_macro_extension 2
+set route_guide [make_result_file route.guide]
+global_route -guide_file $route_guide \
+               -congestion_iterations 100 \
+               -verbose
+
+source ../../../platforms/sky130hd/setRC.tcl
+set_propagated_clock [all_clocks]
+estimate_parasitics -global_routing
+
+#detail_route
+set_thread_count 2
```

Comment:
Why hard-coded to 2?

Reply:
@rovinski Copied from OR unit tests. Maybe we can add `#increase number of cores based on availability`
Is that ok?

Reply:
If it's copied then why can't it be linked or symlinked?